transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -746,9 +746,10 @@ protected void flush0() {
             if (!isActive()) {
                 try {
                     if (isOpen()) {
-                        outboundBuffer.failFlushed(NOT_YET_CONNECTED_EXCEPTION);
+                        outboundBuffer.failFlushed(NOT_YET_CONNECTED_EXCEPTION, true);
                     } else {
-                        outboundBuffer.failFlushed(CLOSED_CHANNEL_EXCEPTION);
+                        // Not notify as the channel is closed already.
```

Comment:
'Do not trigger channelWritabilityChanged because the channel is closed already.'